### ISSUE TITLE
Calendar keyboard navigation: inline calendar Shift + Tab does not work.

### ISSUE THREAD

**Issue by jennylgh** [primefaces/primevue#2951]
### Describe the bug

When calendar inner focusable elements receives focus, Shift + Tab does not work.

There is a [similar issue created](https://github.com/primefaces/primevue/issues/2640). But it's closed with comments that it will be fixed in later version. As of version 3.4.17, issue is still reproducible 

### Reproducer

https://codesandbox.io/s/wizardly-wave-wj1yx8?file=/src/App.vue

### PrimeVue version

3.4.17

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vue CLI App

### Browser(s)

ALL

### Steps to reproduce the behavior

Go to [CodeSandbax demo page:](https://codesandbox.io/s/wizardly-wave-wj1yx8?file=/src/App.vue)
1. Start tabbing until focus comes to calendar header Prev Month icon
2. Keep pressing Shift + Tab, observe next element that has focus.

### Expected behavior

Focus should return to previous focusable tags.

Actual: focus is stuck inside calendar.

**Comment by tugcekucukoglu** [comment]
Could you please try to use the latest PrimeVue version? Calendar accessibility is updated https://github.com/primefaces/primevue/issues/2838
